cmd/queuecontroller/app/options.go

```diff
@@ -5,14 +5,31 @@ package app
 
 import (
 	"flag"
+
+	kaiflags "github.com/NVIDIA/KAI-scheduler/pkg/common/flags"
+)
+
+const (
+	defaultSchedulingQueueLabelKey = "kai.scheduler/queue"
+	defaultMetricsNamespace        = "kai"
```

Comment:
The default namespace should be "kai-scheduler"

Reply:
Please validate against the current default kai namespace

Reply:
I pushed a fix for that - DefaultMetricsNamespace as a const in common